README.md

```diff
@@ -1,87 +1,69 @@
-<img alt="Drupal Logo" src="https://www.drupal.org/files/Wordmark_blue_RGB.png" height="60px">
+# Setup
 
-Drupal is an open source content management platform supporting a variety of
-websites ranging from personal weblogs to large community-driven websites. For
-more information, visit the Drupal website, [Drupal.org][Drupal.org], and join
-the [Drupal community][Drupal community].
+## Local setup.
 
-## Contributing
-
-Drupal is developed on [Drupal.org][Drupal.org], the home of the international
-Drupal community since 2001!
-
-<<<<<<< HEAD
 Copy development services definitions
 ```
 cp web/sites/development.services.yml web/sites/default/services.local.yml
 ```
 
+Local settings (sites/default/settings.local.php)
+```php
+<?php
+
+/**
+ * Database connection.
+ */
+$databases['default']['default'] = [
+ 'database' => getenv('DATABASE_DATABASE') ?: 'db',
+ 'username' => getenv('DATABASE_USERNAME') ?: 'db',
+ 'password' => getenv('DATABASE_PASSWORD') ?: 'db',
+ 'host' => getenv('DATABASE_HOST') ?: 'mariadb',
+ 'port' => getenv('DATABASE_PORT') ?: '',
+ 'driver' => getenv('DATABASE_DRIVER') ?: 'mysql',
+ 'prefix' => '',
+];
+
+$settings['config_sync_directory'] = '../config/sync';
```

Comment:
I think this should bet set in `settings.php` (too).

Reply:
Added